src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -1789,9 +1789,8 @@ void FlexPA::revertAccessPoints()
 {
   const auto& unique = unique_insts_.getUnique();
   for (auto& inst : unique) {
-    const dbTransform xform = inst->getTransform();
-    const Point offset(xform.getOffset());
-    dbTransform revertXform(Point(-offset.getX(), -offset.getY()));
+    dbTransform revertXform;
```

Comment:
follow naming standards
```suggestion
    dbTransform revert_transform;
```